src/main/java/com/vailsys/freeclimb/api/call/CreateCallRequest.java

```diff
@@ -16,21 +16,20 @@
 class CreateCallRequest {
 	public String to;
 	public String from;
-
 	public String applicationId;
-
 	public CallOptions options;
 
 	/**
-	 * Setup a new call creation payload using an applicationId.
+	 * Setup a new call creation payload.
 	 *
 	 * @param to            The number to call out to (DNIS). This can be any valid
 	 *                      phone number formatted in E.164 format.
 	 * @param from          The number to call from (ANI). This must be a number
 	 *                      purchase from FreeClimb or a verified phone number owned
 	 *                      by the user.
 	 * @param applicationId The {@code applicationId} for the registered FreeClimb
-	 *                      application which should handle this call.
+	 *                      application which should handle this call. Required if
+	 *                      no `parentCallId` or `callConnectUrl` has been provided.
```

Comment:
Since the strategy is to have the user provide `null` when they are not using an applicationId, I think this will produce a JSON object with applicationId set to `null` like

```json
{
  "applicationId": null,
}
```

Have we verified that this does not fail a schema check in FreeClimb's API? I feel like it's expecting it to have an applicationId or for the entire field not to be present. I'm not sure it will accept `null`

Reply:
It appears that removing the applicationId field also fails the schema check so I've updated the description.